### PR TITLE
Add tile usage policy

### PR DESCRIPTION
**Pull request by ZeLonewolf**
Adds a usage policy and link to my AWS tile server.

### CODE REVIEW

**Review by 1ec5 — COMMENTED** [review]
I didn’t see anything in the policy that struck me as a mistake. However, I think the policy and readme should clarify that the policy is scoped to usage of the tiles, irrespective of use of the style or application being developed in this repository. That is, someone could use the tileset to create their own custom map style for a personal project, or someone could use the OSM Americana style but point it to their own tile server, even for a commercial endeavor.

File: TILE_USAGE.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+**Effective Date: June 12, 2023**
+
+Thank you for using the OpenStreetMap Americana [Community Vector Tile Server](https://tile.ourmap.us) (referred to as "the Server"). By accessing and using the tiles provided by the Server, you agree to comply with the following Tile Usage Policy:
+
+1. Purpose:
+   The Server aims to support hobbyists, small community groups, and individuals who require vector tiles for their projects or development of custom map styles. The Server is a community resource provided at no cost, but limitations may apply based on server capacity and financial constraints.
+
+2. Attribution:
+   When using tiles from the Server, you must provide proper attribution to OpenStreetMap and OpenMapTiles as required by their respective attribution policies.
```
Comment (akadouri) [inline TILE_USAGE.md:13]:
```suggestion
   When using tiles from the Server, you must provide proper attribution to [OpenStreetMap](https://wiki.osmfoundation.org/wiki/Licence/Attribution_Guidelines#OpenStreetMap_Attribution_Guideline) and [OpenMapTiles](https://github.com/openmaptiles/openmaptiles/blob/master/LICENSE.md) as required by their respective attribution policies.
```

Could hyperlink to the policies here

**Comment by ZeLonewolf** [comment]
Okay, I think I've captured this sentiment - see if that scratches the itch?

File: README.md [hunk 2]
```diff
@@ -71,4 +71,4 @@ We are hoping that it will support more countries; you can [help us](https://git
 
 ## Tile Server
 
-Americana's GitHub map uses a privately-donated community [tile server](https://tile.ourmap.us). It is available for use by other hobbyist and community projects. See also the [tile server usage policy](TILE_USAGE.md).
+While the source code in this repository remains dedicated to the public domain under a CC0 waiver, Americana's GitHub map uses a privately-donated community [tile server](https://tile.ourmap.us) to run the live map. This server is available for use by other hobbyist and community projects with constraints. See the [tile server usage policy](TILE_USAGE.md) for more details. This usage policy describes the allowable usage of the tiles for other projects, separate from this style or the application being developed in this repository.
```
Comment (1ec5) [inline README.md:74]: The use of this tile server is actually just a default:

```suggestion
Although the source code in this repository is dedicated to the public domain under a CC0 waiver, it is configured by default to load map tiles from a privately-donated community [tile server](https://tile.ourmap.us). This server is available for use by other hobbyist and community projects with constraints. See the [tile server usage policy](TILE_USAGE.md) for more details. This usage policy describes the allowable usage of the tiles for other projects, separate from this style or the application being developed in this repository.
```

Consider adding something here in the configuration guide alongside the section about MapTiler:

https://github.com/ZeLonewolf/openstreetmap-americana/blob/aba811e5aff7b075ba06d8bab19fe81657a0aea2/CONTRIBUTING.md#L104-L109

**Comment by ZeLonewolf** [comment]
I added additional links to how to run your own tile server in the section next to the MapTiler discussion. Someone that wants to run this style has the option of paying a commercial provider or rolling their own and the guide should reflect that.

**Review by 1ec5 — APPROVED** [review]
Seems reasonable to me.

**Pull request merged**